frontend/src/ResourceTreeToolbar.svelte

```diff
@@ -212,6 +213,14 @@
           modifierView = SearchView;
         },
       },
+
+      {
+        text: "Show Script",
+        iconUrl: "/icons/identify.svg",
```

Comment:
Search view directly above this uses this icon. Identify also uses the same icon. It's worth modifying one of the existing ones. Perhaps the "new" icon, but change the plus to some horizontal lines?

If there isn't time for that, I'd at least use a different icon from this one.